Bottom: b053011fc79ad5b43c47ad140e990e7f99a120de
Top:    d52e563764ba9b50ed83e1792954bd39b07b266d
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-15 13:38:58 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/include/linux/net.h b/include/linux/net.h
index af94770..d2f9830 100644
--- a/include/linux/net.h
+++ b/include/linux/net.h
@@ -185,7 +185,7 @@ struct proto_ops {
 	ssize_t 	(*splice_read)(struct socket *sock,  loff_t *ppos,
 				       struct pipe_inode_info *pipe, size_t len, unsigned int flags);
 	int		(*set_peek_off)(struct sock *sk, int val);
-        int		(*peek_len) (struct socket *sock);
+	int		(*peek_len) (struct socket *sock);
 };
 
 #define DECLARE_SOCKADDR(type, dst, src)	\
